storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -58,12 +58,19 @@ Use a Field Set to group checkboxes.
 Please note: [NVDA has a bug](https://github.com/nvaccess/nvda/issues/12718) which causes it to
 not report a description connected to a Field Set when it contains checkboxes.
 
-Try to avoid using descriptions for Field Sets containing checkboxes for this reason.
+For this reason, we add the description text to the label.
+Screen reader users will no longer be able to skip the description,
+but we think that is preferable to missing the description entirely.
+Add an `id` to the Field Set and the description, and add both to the `aria-labelledby` attribute of Field Set.
```

Comment:
I’m thinking we should add a snippet with the essential code for this. Just something like:

```
<Fieldset id="fieldset" labelledby="fieldset description" legend="Label">
  <Paragraph id="description">Description
```

This makes it crystal clear for people what markup to add for this guidelines. Currently, people must either read these instructions very closely or compare code of multiple examples and pick what they need.